//package candy.protocol;

message MseFuncSwich
{
	optional int32 time_now = 4;
	repeated int32 index    = 1;
	repeated bool  enable   = 2;	
  repeated int32  level    = 3;
  repeated int32 end_time = 5;
  optional string str     = 6[default=""];
  optional string start_time = 7;
  repeated int32 is_show = 8;
}